### PR TITLE
Add Python3 support

### PR DESCRIPTION
**Pull request by kurokarazu**
Fixed some problem running in Python3.

### CODE REVIEW

File: pyhgvs/utils.py [hunk 1]
```diff
@@ -2,7 +2,11 @@
 Helper functions.
 """
 
-from itertools import imap
+try:
+    from itertools import imap
+except ImportError:
+    # Python 3
+    imap=map
```
Comment (naegelyd) [inline pyhgvs/utils.py]: @lucaswiman Is this OK with you or would you prefer `from six.moves import map as imap`?

File: pyhgvs/__init__.py [hunk 2]
```diff
@@ -1201,7 +1201,7 @@ def get_ref_alt(self, is_forward_strand=True):
         # Represent duplications are inserts.
         if self.mutation_type == "dup":
             alleles[0] = ""
-            alleles[1] = alleles[1][:len(alleles[1]) / 2]
+            alleles[1] = alleles[1][:int(len(alleles[1]) / 2)]
```
Comment (lucaswiman) [inline pyhgvs/__init__.py]: `len(alleles[1]) // 2` would be more concise and probably a bit faster.

File: pyhgvs/utils.py (hunk 1 above)
Comment (lucaswiman) [reply pyhgvs/utils.py]: No reason to add `six` if this is the only place it's needed. I'd prefer to have python 3 be the "default" python and, like:
```python
try:
     from itertools import imap as map
except ImportError:  # python 3
    pass

...

for trans in map(make_transcript, read_refgene(refgene_file)):
```

**Comment by naegelyd** [comment]
@kurokarazu Thanks for your contributions here. If you make the edits @lucaswiman, I think we can get this merged next week. Thanks again.

**Comment by kurokarazu** [comment]
@lucaswiman <https://github.com/lucaswiman> Thanks for your comments. I
agree your modifications here.

2017-10-14 9:00 GMT+08:00 Don Naegely <notifications@github.com>:

**Comment by jtratner** [comment]
Thanks for the PR - we added this separately as per #42 (ish)

**Pull request closed without merging**